_posts/2015-05-23-photoacoustic-penetration.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Hearing with light: photoacoustic radiology improved"
+author: RaoOfPhysics
+---
+
+_Using interstitial light source for tissue-imaging technique based on photoacoustic spectroscopy improves penetration depth and shows promise for minimally invasive radiology_
+
+Photoacoustics (PA) involves shining modulated light (*photo–*) on a material, causing the substance to undergo rapid thermal expansion and contraction in such a way that the pressure differences manifest as sound waves (*acoustics*)[^1]. Materials vary in the sounds they produce corresponding to light at different wavelengths. A light-vs-sound spectrum for assorted materials can be created by “listening” to the pressure waves produced in that material by light shone on it at varying wavelengths. The phenomenon was discovered by Alexander Graham Bell in the late 19th Century, but has found use in modern materials science relatively recently.
+
+PA imaging provides a guiding mechanism for medical procedures without the risks of ionising radiation and the cost of expensive equipment. Light produced by lasers is shone on photoabsorbers, such as haemoglobin, which produce pressure waves that are detected by transducers to produce images of the local region. PA also provides high-contrast images of the surgical tools being used in the medical procedure.
```

Comment:
"produce" three times in a sentence. How about "Molecules in the body such as the blood's haemoglobin can absord light produced by lasers. The resulting pressure waves are detected by transducers and translated into visual images of the targeted region"?